### ISSUE TITLE
FileStorageTier and ReFS on Windows 10

### ISSUE THREAD

**Issue by i3v** [MicrosoftDocs/windows-powershell-docs#3027]
I've just created a test tiered storage space on a single Windows 10 workstation with 3 SSD drives and 2HDD drives, and a shingle ReFS formatted volume (`M:`) on it.
It was a complete surprise for me, that the `Get-FileStorageTier` appears to be broken:
```
PS C:\WINDOWS\system32> $vd | Get-StorageTier

FriendlyName            TierClass   MediaType ResiliencySettingName FaultDomainRedundancy     Size FootprintOnPool StorageEfficiency
------------            ---------   --------- --------------------- ---------------------     ---- --------------- -----------------
DataStoreVDisk1-SSDTier Performance SSD       Simple                0                      5.36 TB         5.36 TB           100.00%
DataStoreVDisk1-HDDTier Capacity    HDD       Simple                0                     18.15 TB        18.15 TB           100.00%


PS C:\WINDOWS\system32> Get-FileStorageTier -FilePath "m:\temp\diskspd.junk"
Get-FileStorageTier : The specified volume does not support storage tiers.
Activity ID: {3215d574-93df-003f-8bd6-1532df93d801}
At line:1 char:1
+ Get-FileStorageTier -FilePath "m:\temp\diskspd.junk"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (StorageWMI:ROOT/Microsoft/...FileStorageTier) [Get-FileStorageTier], CimException
    + FullyQualifiedErrorId : StorageWMI 56000,Get-FileStorageTier
```
[This](https://getsysadminblog.com/2016/12/02/set-filestoragetier-fails-on-microsoft-refs-formatted-volume/) article describes the issue, AFAIU. I can confirm that after re-formatting the same volume as NTFS, the `Get-FileStorageTier` works:
```
PS C:\WINDOWS\system32> Get-FileStorageTier -FilePath "M:\temp\diskspd.junk"

FilePath             FileSize    DesiredStorageTierClass FileSizeOnPerformanceTierClass FileSizeOnCapacityTierClass PlacementStatus State   Placement
                                                                                                                                            Percentag
                                                                                                                                            e
--------             --------    ----------------------- ------------------------------ --------------------------- --------------- -----   ---------
M:\temp\diskspd.junk 34359738368 Unknown                 34359738368                    0                           Unknown         Unknown 0.00%
```

Please correct me if I'm wrong, but this looks like a long-standing major issue, which effectively makes ReFS unusable with tiered storage spaces (at least on Windows 10, not sure about Server versions). 

However, it looks like this issue is not officially documented anywhere. In particular, the `Get-FileStorageTier` documentation does not mention that the command would just fail on ReFS on Windows 10.
The error message printed by the cmdlet is misleading as well.

So... Could you please add a big red note "ReFS on a Windows 10 is not supported" to the `Get-FileStorageTier` documentation, at least?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 20d99fe3-cfa7-f520-17b1-d10c9cc4b3b6
* Version Independent ID: d6f0d561-f948-e558-db12-7c47b4bd175c
* Content: [Get-FileStorageTier (Storage)](https://docs.microsoft.com/en-us/powershell/module/storage/get-filestoragetier?view=windowsserver2019-ps)
* Content Source: [docset/winserver2019-ps/storage/Get-FileStorageTier.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/main/docset/winserver2019-ps/storage/Get-FileStorageTier.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend**

**Comment by RAJU2529** [comment]
@i3v ReFs storage feature supported on Windows 10 Workstation OS.
Did you create a new volume  M  and created temp folder in M.

**Comment by i3v** [comment]
@RAJU2529 ,
Yes, sure, I've created volume "M:".

You probably misread what I was talking about:
> So... Could you please add a big red note "ReFS on a Windows 10 is not supported" _to the Get-FileStorageTier documentation_, at least?

I was not saying that ReFs is not supported on Windows 10. It is supported, partially. My point was that `Get-FileStorageTier`+`ReFs`+`Windows10` combination is  is not supported.

**Comment by RAJU2529** [comment]
@i3v . Did you using windows 10 for  workstation os.

**Comment by i3v** [comment]
> Windows 10 Workstation OS

I did that test on Windows 10 Enterprise edition. And the Enterprise edition [got](https://en.wikipedia.org/w/index.php?title=Windows_10_editions&oldid=1119797865) all the features of the "Pro for Workstations" edition.

**Comment by RAJU2529** [comment]
@i3v, ReFS  file system supported only on Pro for Workstations

**Comment by i3v** [comment]
> ReFS file system supported only on Pro for Workstations

That's not true. For example [here](https://learn.microsoft.com/en-us/windows/deployment/planning/windows-10-removed-features) they say:
> <html>
<body>


esilient File System (ReFS) (added: August 17, 2017) | Creation ability will be available in the following editions only: Windows 10 Enterprise and Windows 10 Pro for Workstations. Creation ability will be removed from all other editions. All other editions will have Read and Write ability. | 1709
-- | -- | --



</body>
</html>

That is, ReFS is supported on Windows 10 Enterprise.
